.github/workflows/ci.yml

```
@@ -0,0 +1,65 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
```

This lets us test feature branches on forks, and also trigger builds via the GitHub UI:

```suggestion
on: [push, pull_request, workflow_dispatch]
```